platform/net/interface_configuration_creator_test.go

```diff
@@ -239,6 +267,47 @@ func describeInterfaceConfigurationCreator() {
 					})
 				})
 			})
+
+			Context("when static network has alias, dhcp network is not allowd to have alias", func() {
```

Comment:
Can you include some links to the reason for this? I imagine there is a IETF RFC discussing this. Just add a link to section for this rule. Thx.